app/services/sensors/sensor_service.rb

```diff
@@ -0,0 +1,39 @@
+module Sensors
+  class SensorService
+    def initialize sensor
+      @sensor = sensor
+    end
+
+    def get_stats
+      {
+        total: Sensor.count,
+        active: Sensor.where(status: :active).count,
+        inactive: Sensor.where(status: :inactive).count,
+        faulty: Sensor.where(status: :error).count
```

Comment:
```suggestion
        active: Sensor.active.count,
        inactive: Sensor.inactive.count,
        faulty: Sensor.error.count
```
Dùng hàm hỗ trợ của enum